Documentation/tls/tls-certificates.md

```diff
@@ -28,51 +28,14 @@ TLS certs may not be enabled for:
 * Pod to Pod communication
 * Ingress to Pod communication
 
-
 For more information, see the Kubernetes document: [Manage TLS Certificates in a Cluster][manage-tls].
 
-## Providing a TLS certification for Tectonic Ingress
-
-First, comment out the existing self-signed Ingress TLS in your platform. For example: `platforms/aws/tectonic.tf`.
-
-```
-/*
-module "ingress_certs" {
-  source = "../../modules/tls/ingress/self-signed"
-
-  base_address = "${module.masters.ingress_internal_fqdn}"
-  ca_cert_pem  = "${module.kube_certs.ca_cert_pem}"
-  ca_key_alg   = "${module.kube_certs.ca_key_alg}"
-  ca_key_pem   = "${module.kube_certs.ca_key_pem}"
-}
-*/
-```
-
-Then, configure the user provided certificates in your platform:
-
-```
-module "ingress_certs" {
-  source = "../../modules/tls/ingress/user-provided"
-
-  ca_cert_pem = <<EOF
------BEGIN CERTIFICATE-----
-<contents of the public CA certificate in PEM format>
------END CERTIFICATE-----
-EOF
+## Creating custom TLS certificates
 
-  cert_pem = <<EOF
------BEGIN CERTIFICATE-----
-<contents of the public ingress certificate signed by the above CA in PEM format>
------END CERTIFICATE-----
-EOF
+Tectonic Installer provides a package containing a sample Terraform project to generate user provided certificates. It is meant to be a starter for customizations.
```

Comment:
I'd prefer we not link to this from our official product docs, or if we do it should contain a disclaimer that this is a non-supported reference implementation only. Caution and close inspection of source code is advised when generating any certs for production use.